t/t0402-block-command-on-gvfs.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+
+test_description='block commands in GVFS repo'
+
+. ./test-lib.sh
+
+test_expect_success 'test fsck' '
+	git config alias.kcsf fsck &&
+	git config core.gvfs true &&
+	test_must_fail git fsck &&
+	test_must_fail git kcsf &&
+	git config --unset core.gvfs &&
+	test_must_fail git -c core.gvfs=true fsck &&
+	test_must_fail git -c core.gvfs=true kcsf
+'
+
+test_expect_success 'test gc' '
+	git config alias.cg gc &&
+	git config core.gvfs true &&
+	test_must_fail git gc &&
+	test_must_fail git cg &&
+	git config --unset core.gvfs &&
+	test_must_fail git -c core.gvfs=true gc &&
+	test_must_fail git -c core.gvfs=true cg
+'
+
+test_expect_success 'test prune' '
+	git config alias.enurp prune &&
+	git config core.gvfs true &&
+	test_must_fail git prune &&
+	test_must_fail git enurp &&
+	git config --unset core.gvfs &&
+	test_must_fail git -c core.gvfs=true prune &&
+	test_must_fail git -c core.gvfs=true enurp
+'
+
+test_expect_success 'test repack' '
+	git config alias.kcaper repack &&
+	git config core.gvfs true &&
+	test_must_fail git repack &&
+	test_must_fail git kcaper &&
+	git config --unset core.gvfs &&
+	test_must_fail git -c core.gvfs=true repack &&
+	test_must_fail git -c core.gvfs=true kcaper
+'
+
+test_expect_success 'test submodule' '
+	git config alias.eludombus submodule &&
+	git config core.gvfs true &&
+	test_must_fail git submodule status &&
+	test_must_fail git eludombus status &&
+	git config --unset core.gvfs &&
+	test_must_fail git -c core.gvfs=true submodule status &&
+	test_must_fail git -c core.gvfs=true eludombus status
```

Comment:
This is *a lot* of repeated code. It would make more sense to refactor this into a function:

```sh
not_with_gvfs () {
    command=$1 &&
    shift &&
    test_expect_success "test $command" "
        test_config alias.g4rbled $command &&
        test_config core.gvfs true &&
        test_must_fail git $command $* &&
        test_must_fail git g4rbled $* &&
        test_unconfig core.gvfs &&
        test_must_fail git -c core.gvfs=true $command $* &&
        test_must_fail git -c core.gvfs=true g4rbled $*
    "
}

not_with_gvfs fsck
not_with_gvfs gc
not_with_gvfs prune
not_with_gvfs repack
not_with_gvfs submodule status
```

Reply:
That is much more clever but it will really reduce my LOC metrics. :)  Thanks for the better way.

Reply:
> it will really reduce my LOC metrics.

🤣